tools/jsonschema2go/jsonschema.go

```diff
@@ -661,7 +661,7 @@ func (subSchema *JsonSubSchema) link(job *Job) (err error) {
 	if ref := subSchema.Ref; ref != nil && *ref != "" {
 		subSchema.RefSubSchema = job.result.SchemaSet.all[subSchema.RefSchemaURL]
 		if subSchema.RefSubSchema == nil {
-			return fmt.Errorf("Subschema %v not loaded when updating %v", subSchema.RefSchemaURL, subSchema.SourceURL)
+			return fmt.Errorf("subschema %v not loaded when updating %v", subSchema.RefSchemaURL, subSchema.SourceURL)
```

Comment:
u no like uppercase? :)

Reply:
let's hope no-one is doing case-sensitive checks for those errors 😬

Reply:
Haha, it fails the staticcheck :)

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
good to know, thanks